dask_kubernetes/operator/tests/test_operator.py

```diff
@@ -60,12 +60,29 @@ async def test_simplecluster(k8s_cluster, kopf_runner, gen_cluster):
         async with gen_cluster() as cluster_name:
             # TODO test our cluster here
             scheduler_pod_name = "simple-cluster-scheduler"
-            # scheduler_service_name = "simple-cluster"
+            worker_pod_name = "simple-cluster-worker-1"
             while scheduler_pod_name not in k8s_cluster.kubectl("get", "pods"):
                 await asyncio.sleep(0.1)
+            while cluster_name not in k8s_cluster.kubectl("get", "svc"):
+                await asyncio.sleep(0.1)
+            while worker_pod_name not in k8s_cluster.kubectl("get", "pods"):
+                await asyncio.sleep(0.1)
+
+            from dask.distributed import Client
+
+            with k8s_cluster.port_forward(f"service/{cluster_name}", 8786) as port:
```

Comment:
The port-forwarding is failing as well. This is because the service cannot be found. I added time.sleep before creating the client, but the issue still persisted.

Reply:
You probably need `asyncio.sleep` to allow the event loop to keep running.